### PR TITLE
Landing page beauty

### PR DESCRIPTION
**Pull request by amsmyth1**
Add random photo urls to populate on the landing page. 

Add some styling on landing page and color variables to style sheet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,8 +93,10 @@ These instructions will get you a copy of the project up and running on your loc
 
   6. Inside of the application.yml create the environment variables for your google client_id and client_secret as. At the bottom of the file enter the following:
   ```
-  GOOGLE_CLIENT_ID: <your client_id goes here>
-  GOOGLE_CLIENT_SECRET: <your client_secret goes here>
+  GOOGLE_CLIENT_ID: '13023404400-n1orn9rntvifnqffcj45rbji69b7bq1v.apps.googleusercontent.com'
+  GOOGLE_CLIENT_SECRET: 'zx8ecS0uBkPAMmh207pIPIZE'
+  ys_engine_url: 'http://localhost:3001'
+  mapquest_key: 'SHKnstA5dZDGGsyGX1AkAan6iEbUWUE8'
```
Comment (abreaux26) [inline README.md]: I don't think we want to expose our stuff like this right?
I like how you had it before of `<your client_id goes here>`

Comment (amsmyth1) [reply README.md]: DUH! Great catch!!! Sorry about that was not thinking.

**Review by jbranham1 — APPROVED** [review]
Looks so good!
Adding this color to my stuff!

**Review by abreaux26 — APPROVED** [review]
This looks really good

**Pull request merged**